seagulls-space-shooter-demo/src/seagulls/space_shooter/_laser.py

```diff
@@ -40,7 +42,6 @@ def render(self, surface: Surface) -> None:
 
         surface.blit(laser_sprite, self._position)
 
-    @lru_cache()
     def _get_cached_laser(self) -> Surface:
```

Comment:
should we rename this method now that it's not cached? maybe clarify a bit more and call it `_get_sprite()`?

Reply:
Yep, updated